### DISCUSSION TITLE
Wechselrichter Firmware Update (HM 1500)

### DISCUSSION THREAD

**Discussion by BigT71** [tbnobody/OpenDTU#988] [General]
Hallo allerseits,

ist es möglich oder irgendwann geplant per OpenDTU für den Wechselrichter selbst ein Firmware Update durchzuführen?

Oder ist es nur mit der Original DTU möglich?

Grüße

**Comment by bitboy00** [comment]
Bisher scheint es nur mit der originalen DTU zu gehen.

**Comment by BerniXS** [comment]
Moin zusammen,
gibt es zu diesem FW für die Hoymiles Wechselrichter schon neuere Informationen oder bleibt es bei der originalen DTU?

VG
